hicexplorer/lib/viewpoint.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+
+import logging
+log = logging.getLogger(__name__)
+
+
+class Viewpoint():
+
+    def __init__(self, pHiCMatrix=None):
+        self.hicMatrix = pHiCMatrix
+
+    def setHiCMatrixObj(self, pHiCMatrix):
```

Comment:
why do you need a set()?
https://docs.python.org/3/library/functions.html#property